quartz/components/scripts/graph.inline.ts

```diff
@@ -182,23 +182,17 @@ async function renderGraph(container: string, fullSlug: FullSlug) {
     "--darkgray",
     "--bodyFont",
   ] as const
-  const computedStyleMap = cssVars.reduce(
-    (acc, key) => {
-      acc[key] = getComputedStyle(document.documentElement).getPropertyValue(key)
-      return acc
-    },
-    {} as Record<(typeof cssVars)[number], string>,
-  )
 
-  // calculate color
+  // Dynamic color function
```

Comment:
this is probably unnecessary and we should avoid recomputing from the dom when possible, we already recompute this style map on calling renderGraph again